tests/driver_si70xx/Makefile

```diff
@@ -1,6 +1,8 @@
 include ../Makefile.tests_common
 
-# This test should also work with Si7006, Si7013 and Si7020 variants.
-USEMODULE += si7021
+# This test should also work with si7006, si7013, si7020 and si705x variants.
```

Comment:
```suggestion
# This test should also work with Si7006, Si7013, Si7020 and Si705x variants.
```

Reply:
I was going with all lowercase when referring to the RIOT module name, and upper case when referring to the chip itself. Here since this is the string you need to pass to the Makefile I'd prefer to leave it lowercase.

Reply:
Sounds OK.